### DISCUSSION TITLE
Multiple selection slow

### DISCUSSION THREAD

**Discussion by nicktan147** [awcodes/filament-curator#433] [Q&A]
**My Environment**

```
Windows 11
PHP8.1
Nginx
Filament-Starter
Filament Curator v3.49
```

Here's the vid
[screen-capture.webm](https://github.com/awcodes/filament-curator/assets/7764638/2d0ce616-397c-4f75-9fd8-46ab5e12692a)

When I use multiple selection selecting multiple images, it will become very slow for every picture clicked, we notice the behavior it will call livewire and redownload all images again, we suspect that could make it slow. Does anyone face similar issue?

**Comment by SeiberPro** [comment]
Can you show blade where you tied this field using wire:model

**Reply by awcodes** [reply]
What? 

There is no wire:model though.

😀

**Reply by SeiberPro** [reply]
and in general, the blade template where can I see the file upload?

**Reply by awcodes** [reply]
The file upload is a filament form. All of this functionality is in the CuratorPanel class though. Start there.
